server/src/database/migrations/20230718070849_alter_table_user_and_add_table_user_settings/migration.sql

```diff
@@ -0,0 +1,47 @@
+-- Temporarily rename 'locale' to 'old_locale' to keep its values
+ALTER TABLE "users" RENAME COLUMN "locale" TO "old_locale";
+
+-- Adding 'settingsId' to 'users' table without NOT NULL constraint initially
+ALTER TABLE "users" ADD COLUMN "settingsId" TEXT;
+
+-- Creating 'user_settings' table
+CREATE TYPE "ColorScheme" AS ENUM ('Light', 'Dark', 'System');
```

Comment:
@lucasbordeau @charlesBochet I don't think it's good to remove the data migration part, is will cause inconsistency problems because `User` must have relation with `UserSettings`, so if data are not migrated they are not going to be synced with the schema.